Content.Server/ReagentOnItem/EntitySystems/ReagentOnItemSystem.cs

```diff
@@ -0,0 +1,91 @@
+using Content.Shared.Chemistry.Components;
+using Content.Shared.Item;
+using Content.Shared.Popups;
+using Content.Shared.ReagentOnItem;
+using Content.Server.Fluids.EntitySystems;
+using Content.Shared.Clothing.Components;
+using Content.Shared.IdentityManagement;
+
+namespace Content.Server.ReagentOnItem;
+
+/// <summary>
+///     The system that helps deal with a genaric reagent being applied to items.
+/// </summary>
+public sealed class ReagentOnItemSystem : EntitySystem
+{
+    [Dependency] private readonly PuddleSystem _puddle = default!;
+    [Dependency] private readonly SharedPopupSystem _popup = default!;
+    public override void Initialize()
+    {
+        base.Initialize();
+    }
+
+    /// <summary>
+    ///     The function tries to apply the given solution mixture to the item.
+    ///     It will apply all reagents that can stick and will spill anything
+    ///     that cannot stick to the floor.
+    /// </summary>
+    /// <returns> Returns false if there is no solution or if entity isn't an item and true otherwise. </returns>
```

Comment:
```suggestion
    /// <returns> False if there is no solution or if entity isn't an item and true otherwise. </returns>
```
You can omit the returns since returns is the section it's in.